docs/internal_api/index.rst

```diff
@@ -138,29 +148,63 @@ Lines
 
    grid.lines._angle_of_2_vectors
 
-IO
-==
 
-Ugrid
+Grid Parsing and Encoding
+=========================
+
+UGRID
 -----
 .. autosummary::
    :toctree: _autosummary
 
+   io._ugrid._read_ugrid
+   io._ugrid._encode_ugrid
+   io._ugrid._standardize_fill_value
    io._ugrid._is_ugrid
 
-Utils
+MPAS
+----
+.. autosummary::
+   :toctree: _autosummary
+
+   io._mpas._read_mpas
+   io._mpas._primal_to_ugrid
+   io._mpas._dual_to_ugrid
+   io._mpas._set_global_attrs
+   io._mpas._replace_padding
+   io._mpas._replace_zeros
+   io._mpas.__to_zero_index
+
+
+EXODUS II
```

Comment:
I think it is widely used as Exodus (even though technically Exodus II should be correct). I am not sure if we should go the "II" route.

Reply:
Dropping the "II" makes sense.